### DISCUSSION TITLE
python-vscode need class-arguments-auto-complete feature

### DISCUSSION THREAD

**Discussion by yingshaoxo** [microsoft/pylance-release#4111] [Enhancement]
In typescript-vscode, the extension will help you auto-fill initialization arguments for class.

But in Python, I didn't see it.

___

typescript vscode:

<img width="1089" alt="image" src="https://user-images.githubusercontent.com/17190829/226094888-59ea51c8-1ffa-4d1c-a9af-31bfdc1c3d3f.png">

<img width="968" alt="image" src="https://user-images.githubusercontent.com/17190829/226094903-0dd07aa9-59dc-4733-9fbe-459dc1116345.png">

<img width="309" alt="image" src="https://user-images.githubusercontent.com/17190829/226094908-3988a8dc-8aab-4f3d-a249-091acef411dd.png">

```typescript
export class An_Object {
    id: number | null = null;
    name: string | null = null;
    description: string | null = null;
    likes: number | null = null;
    dislikes: number | null = null;
    alternative_id_list: number[] | null = null;
}
```

___

python-vscode:

<img width="1102" alt="image" src="https://user-images.githubusercontent.com/17190829/226095042-2847e4f0-a679-4432-a03b-c9a850ea7319.png">

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
